cves/CVE-2011-3944.yml

```diff
@@ -276,8 +276,8 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: false
+  note: This bug is about corrupt files.
```

Comment:
It's about reading and writing from corrupt files, so maybe we could argue that there is some ipc element in it?

Reply:
Changed to:
This bug is about reading and writing from corrupt files. Although it is not traditionally IPC, it is still important to note.